src/index.jsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
```

Comment:
also would be good to extract /utils/spotify.js with all dependencies to a separate folder. For example, /spotify-client/. and add index.js with needed exports